tests/xtro-sharpie/iOS-QuickLook.ignore

```diff
@@ -1,2 +1,5 @@
 ## fixed for XAMCORE_4_0
 !incorrect-protocol-member! QLPreviewItem::previewItemTitle is OPTIONAL and should NOT be abstract
+
+## Getting "[FAIL] Selector not found for Foundation.NSUrl : previewItemTitle" when NSUrl conforms to 'QLPreviewItem'
+!missing-protocol-conformance! NSURL should conform to QLPreviewItem (defined in 'QLPreviewConvenienceAdditions' category)
```

Comment:
That's probably because of the entry just above - if it's `optional` then having it not responding is _normal_